src/lib/network/network_autoyast.rb

```diff
@@ -82,13 +82,14 @@ def set_network_service
 
       if use_network_manager && Lan.yast_config.backend.available?
         log.info("- using NetworkManager")
+        NetworkService.use_network_manager
```

Comment:
If you want, you can have `NetworkService.use` so just write: `NetworkService.use(Last.yast_config&.backend)`.

Reply:
I left it as it is just to not touch yast2 having to add the method there and having to update the dependency. The benefit is minimal, so I think we could omit adding it by now maintaining the explicit use calls.

Reply:
It was needed also when writing the network config. Thus, I have changed my mind adding  `NetworkService#use` as suggested.

https://github.com/yast/yast-yast2/pull/1065